.github/workflows/ci-windows.yml

```diff
@@ -55,12 +55,12 @@ jobs:
           gotestsum --format short-verbose --rerun-fails=5 --packages="./..."  --junitfile TEST-unit.xml -- -timeout=30m
```

Comment:
Does your Windows machine supports make ? 
You would be able to replace this with make test-unit command .

Reply:
I do not think so, and the process to add packages to them is slow. I'd keep it as is, using go tooling.

At some point, I'm realising the makefiles are basically pass-through calling Go tools, and probably adding complexity instead of hiding it.